turbo-cli/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "turbo-cli"
 version = "0.1.0"
 authors = ["Sam Wilson <sam.wilson@mesh.xyz>"]
 edition = "2018"
+license = "Apache-2.0"
```

Comment:
Not an Apache-2.0/MIT guy?

Reply:
I just went off the sentry's license. I don't mind changing.